packages/chakra-ui-docs/docs/tabs.mdx

```diff
@@ -1 +1,339 @@
+|           | `role="tablist"`   | Indicates that it's a tablist                                               |
+|           | `aria-labelledby`  | Set to the `id` of the `Tab` that labels the `TabPanel`.                    |
+
+## Props
+
+### Tabs Props
+
+Tabs composes `Box` so you call pass all `Box` related props.
+
+| Name           | Type                                                                              | Default      | Description                                                                                                                     |
+| -------------- | --------------------------------------------------------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| `index`        | `number`                                                                          |              | The controlled index of the tabs.                                                                                               |
+| `defaultIndex` | `number`                                                                          |              | The index of the initial active tab.                                                                                            |
+| `isManual`     | `boolean`                                                                         |              | If `true`, keyboard navigation changes focus between tabs but doens't activate it. User will have to press `Enter` to active it |
+| `variant`      | `line`,`enclosed`,`enclosed-colored`, `soft-rounded`, `solid-rounded`, `unstyled` | `line`       | The visual style of the tab.                                                                                                    |
+| `variantColor` | `string`                                                                          |              | The primary color to use for the selected variant. Use a color key in `theme.colors`                                            |
+| `size`         | `sm`, `md`, `lg`                                                                  | `md`         | The visual size of the tabs                                                                                                     |
+| `orientation`  | `horizontal`, `vertical`                                                          | `horizontal` | The orientation of the tabs                                                                                                     |
+| `isFitted`     | `boolean`                                                                         |              | If `true`, the tabs will stretch to fill the available space                                                                    |
+
+### Tabs Events
+
+| Name       | Description              |
+| ---------- | ------------------------ |
+| `onChange` | Emits the @change event. |
```

Comment:
The event name is `@change`. Description is "Event emitted when the active tab changes"

Reply:
Oh got it. My bad